spec/requests/api/milestones_spec.rb

```diff
@@ -30,6 +30,13 @@
       json_response['iid'].should == milestone.iid
     end
 
+    it "should return a project milestone by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.